exercises/standard_library_types/iterators5.rs

```diff
@@ -0,0 +1,113 @@
+// iterators5.rs
+
+// Rustling progress is modelled using a hash map. The name of the exercise is
+// the key and the progress is the value. Two counting functions were created
+// to count the number of exercises with a given progress. These counting
+// functions use imperative style for loops. Recreate this counting
+// functionality using iterators.
+// Execute `rustlings hint iterators5` for hints.
+//
+// Make the code compile and the tests pass.
+
+// I AM NOT DONE
+
```

Comment:
Generally, I would leave comments somewhere in this exercise with notes that explain:

- What a "stack" is (aka a vec of hashmaps)
- That you probably shouldn't touch the `_for` methods, but rather implement the iterative version in the other methods
- A delineator above the tests module telling the user not to touch what's below it